gov/governance/execute.gno

```diff
@@ -497,56 +501,125 @@ func handleEmissionChangeDistributionPct(params []string) (
 	int, uint64,
 	int, uint64,
 ) {
-	target01 := strToInt(params[0])
-	pct01 := parseUint(params[1], 10, 64)
+	target01, err := strconv.ParseInt(params[0], 10, 64)
+	if err != nil {
+		panic(err)
+	}
+	pct01, err := strconv.ParseUint(params[1], 10, 64)
+	if err != nil {
+		panic(err)
+	}
 
 	target02 := strToInt(params[2])
-	pct02 := parseUint(params[3], 10, 64)
+	pct02, err := strconv.ParseUint(params[3], 10, 64)
+	if err != nil {
+		panic(err)
+	}
 
 	target03 := strToInt(params[4])
-	pct03 := parseUint(params[5], 10, 64)
+	pct03, err := strconv.ParseUint(params[5], 10, 64)
+	if err != nil {
+		panic(err)
+	}
 
 	target04 := strToInt(params[6])
-	pct04 := parseUint(params[7], 10, 64)
+	pct04, err := strconv.ParseUint(params[7], 10, 64)
+	if err != nil {
+		panic(err)
+	}
 
-	return target01, pct01, target02, pct02, target03, pct03, target04, pct04
+	return int(target01), pct01, target02, pct02, target03, pct03, target04, pct04
 }
 
 func handleGnsSetAvgBlockTimeInMs(params []string) int64 {
-	return int64(parseUint(params[0], 10, 64))
+	res, err := strconv.ParseInt(params[0], 10, 64)
+	if err != nil {
+		panic(err)
+	}
+	return res
 }
 
 func handleGovernanceReconfigure(params []string) (
 	uint64, uint64, uint64, uint64, uint64, uint64, uint64,
 ) {
-	votingStartDelay := parseUint(params[0], 10, 64)
-	votingPeriod := parseUint(params[1], 10, 64)
-	votingWeightSmoothingDuration := parseUint(params[2], 10, 64)
-	quorum := parseUint(params[3], 10, 64)
-	proposalCreationThreshold := parseUint(params[4], 10, 64)
-	executionDelay := parseUint(params[5], 10, 64)
-	executionWindow := parseUint(params[6], 10, 64)
+	votingStartDelay, err := strconv.ParseUint(params[0], 10, 64)
+	if err != nil {
+		panic(err)
+	}
+
+	votingPeriod, err := strconv.ParseUint(params[1], 10, 64)
+	if err != nil {
+		panic(err)
```

Comment:
please do `panic(err.Error())`, its better when we run test using gno test

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/358/commits/f114bc428dd1e91275fc0e70a959f59470292da4